manoseimas/lobbyists/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.conf.urls import include, patterns, url
+
+from manoseimas.lobbyists import views
+
+urlpatterns = patterns(
+    '',
+    url(r'^$', 'manoseimas.lobbyists.views.lobbyists.lobbyist_list'),
```

Comment:
Why `'manoseimas.lobbyists.views.lobbyists.lobbyist_list'` instead of `views.lobbyist_list`?